### PR TITLE
Add some missing tests on text field

### PR DESCRIPTION
**Pull request by corranwebster**
Forgot tests for a couple of attributes.

### CODE REVIEW

**Comment by jwiggins** [comment]
This is good? The tests you added failed. TDD!

**Comment by corranwebster** [comment]
The code was written on a computer where I couldn't easily do testing (didn't particularly want to install EDM on it).  But it _is_ good that it failed, as it looks like there may be an actual bug or two.

**Comment by corranwebster** [comment]
Test cases hopefully now working on all platforms.  This is why we write tests.

File: pyface/fields/i_text_field.py [hunk 1]
```diff
@@ -144,6 +142,7 @@ def _observe_control_editing_finished(self, remove=False):
     # Trait change handlers --------------------------------------------------
 
     def _placeholder_updated(self):
+        print(self.control, self.placeholder)
```
Comment (jwiggins) [inline pyface/fields/i_text_field.py]: Debug droppings

**Pull request merged**